prow/config.yaml

```diff
@@ -130,8 +130,6 @@ branch-protection:
             teams:
             - stage-bots
         utils:
-          required_pull_request_reviews:
-            required_approving_review_count: 1
```

Comment:
Until the branch protector actually runs to enforce this change, Tide will fail to merge any PRs that meet all requirements, but do not have an approving review. This is probably not a big deal though?

Reply:
Probably not a big deal... but duly noted, I'll leave this on /hold until I hear from one of the repo owners